app/Models/SnipeSCIMConfig.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Helper;
+use ArieTimmerman\Laravel\SCIMServer\SCIM\Schema;
+use ArieTimmerman\Laravel\SCIMServer\Attribute\AttributeMapping;
+
+
+class SnipeSCIMConfig extends \ArieTimmerman\Laravel\SCIMServer\SCIMConfig
+{
+    public function __constructor(...$params)
+    {
+        \Log::info("CONSTRUCTOR CALLED!!@!!!! FOr my config thing");
+        self::super(...$params);
```

Comment:
I’m not 100% sure what this does. Can you help me understand? I’ve used `parent::someMethodToOverride()`, but I don’t know what this constructor does.

Reply:
Honestly, this constructor does nothing.

The only thing I can think of about `::super()` is that you’re not allowed to directly invoke a constructor - e.g. `parent::__constructor()` would be considered ‘gauche’ because you aren’t allowed to directly invoke a constructor. Or at least my understanding is that. But, otherwise, they would probably (?) be the same.

Honestly I oughtta just pull the method because it doesn’t do anything. I probably put it in just for troubleshooting while I was struggling with the implementation.